### PR TITLE
feat: Add style on mobile device

### PR DESCRIPTION
**Pull request by kosssi**
Je rapatrie le style de la cozy-bar qui est encore dans cozy-ui, voir la suppression dans la PR suivante https://github.com/cozy/cozy-ui/pull/832

### CODE REVIEW

File: src/styles/bar.css [hunk 1]
```diff
@@ -10,6 +10,17 @@
   font-size: 1rem;
 }
 
+@media (max-width: 64em) {
+  [role=banner] {
```
Comment (CPatchane) [inline src/styles/bar.css]: So, no need to have the double selector anymore? Good thing if so 👍

Comment (kosssi) [reply src/styles/bar.css]: I don't understand why we need double selector before, it's ok in banks with this.

Comment (CPatchane) [reply src/styles/bar.css]: Do you try on a native mobile browser? @GoOz seems to say that it was specifically for that case

Comment (GoOz) [reply src/styles/bar.css]: It was clearly a conflict between Cozy-bar styles and UI styles on mobile browsers when minimal-UI was deployed. 
I don't remember clearly what was the bug but it sure be cool to test this on native browser on mobile indeed.

**Pull request merged**